tor/core/user_interaction.py

```diff
@@ -50,8 +50,8 @@ def process_coc(post, config):
     # instead. If they're actually new, then send a message to slack.
     if result == 1:
         send_to_slack(
-            'u/{} has just accepted the CoC!'.format(
-                post.author.name
+            '<http://www.reddit.com/u/{}|u/{}> has just accepted the CoC!'.format(
+                post.author.name, post.author.name
```

Comment:
We could provide a named argument in the string ({} => {user}) and only include the name once (as a keyword argument) in the format() arguments.

Reply:
That... did not occur to me. Yeah, that seems like a decent change.